sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/src/Generated/Extensions/MockableNeonPostgresResourceGroupResource.cs

```diff
@@ -49,7 +49,7 @@ public virtual NeonOrganizationCollection GetNeonOrganizations()
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>OrganizationResource_Get</description>
+        /// <description>Organizations_get</description>
```

Comment:
it should be `Organizations_get`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/liftrneon/Neon.Postgres.Management/main.tsp#L47